latex/interni/doc_interna/norme_di_progetto/res/sections/01_content.tex

```diff
@@ -212,11 +238,21 @@ \subsubsection{Issue Tracking System\textsubscript{g}}
 \end{itemize}
 Inoltre nella project board vengono registrate delle issue che non richiedono verifica, approvazione o neanche integrazione, con lo scopo di monitorare meglio il lavoro di ogni membro del team.\\
 Queste issue verranno chiuse e archiviate manualmente una volta che avranno terminato la loro utilità, un esempio può essere la seguente issue:\\
-\textbf{diario di bordo 21-11-22}; questa issue non necessità verifica, approvazione o integrazione perchè non è di interesse caricare il file nella repo, però è utile tracciare lo svolgimento della issue
+\textbf{diario di bordo 21-11-22}; questa issue non necessità verifica, approvazione o integrazione perchè non è di interesse caricare il file nella repo, però è utile tracciare lo svolgimento della issue.
+
+
+
+
+\subsubsection{Issue Tracking System\textsubscript{g}}
+Il gruppo utilizza l'issue tracking system\textsubscript{g} di Github\textsubscript{g} per tenere traccia delle issue\textsubscript{g}. 
+Le issue verranno determinate dal responsabile, ma la loro assegnazione verrà effettuata dai membri del gruppo, in base alle loro preferenze, i loro ruoli e alle loro disponibilità temporali.\\
```

Comment:
plurali issues in generale.. Togliere in base alle proprie preferenze e mettere priorità issue,ruoli e disponibilità va bene